src/test/java/net/openhft/chronicle/core/time/SystemTimeProviderTest.java

```diff
@@ -25,14 +25,24 @@
 import org.jetbrains.annotations.NotNull;
 import org.junit.Test;
 
+import java.security.SecureRandom;
 import java.util.concurrent.TimeUnit;
 
 import static org.junit.Assert.assertTrue;
 
 public class SystemTimeProviderTest {
     @Test
     public void currentTimeMicros() throws IllegalStateException {
-        FlakyTestRunner.run(Jvm.isArm() || OS.isWindows() || OS.isMacOSX(), this::doCurrentTimeMicros);
+        // Superflaky
```

Comment:
Perhaps something more descriptive? No idea whether it is a bad or good thing.

Reply:
Yepp. I will add why in the comment.